scripts/lib/libvirt/fixtures/cloud-node1-raid.xml

```diff
@@ -83,4 +83,5 @@
       <address type='pci' slot='0x05'/>
     </memballoon>
   </devices>
+
```

Comment:
This is because the test script for this doesn't check for element equality but compares full strings, so this is needed for the tests to pass.

I know, I know :sob: